whittle/search_sub_networks.py

```diff
@@ -143,13 +188,30 @@ def _objective(
     val_dataloader: DataLoader,
     eval: EvalArgs,
     verbose: bool | None = True,
+    objective_1: str = "val_loss",
+    objective_2: str = "parameters",
+    fine_tuned: bool = False,
 ) -> tuple[float, float]:
     model.select_sub_network(config)
-    val_loss = validate(
-        fabric, model, val_dataloader, max_iters=eval.max_iters, verbose=verbose
-    )
-    num_params = compute_parameters(model)
-    return float(val_loss), num_params
+
```

Comment:
we should have an `assert objective_2 in ['parameters', 'latency', 'flops'] `here